static/assets/css/main.css

```diff
@@ -71,3 +71,18 @@ blockquote li,
 blockquote::before {
   border-color: #d3d3d3; /* $color-mediumgray */
 }
+
+.env-commandline .env-specifier,
+.env-dashboard .env-specifier,
+.env-commandline .env-specifier a
+.env-dashboard .env-specifier a
+{
+  color: #333; /* $color-textblack */
+}
+
+.env-commandline::before{
+  border-color: #333;
+}
+.env-dashboard::before {
+  border-color: #74B1DB;
+}
```

Comment:
Proposed changes to text in the image below:

* How about we change the first sentence to simply be `Add a user to this organization`? If users have indicated the account status is salient to them when making invitations, lmk; I assume they haven't.
* `To invite a user and give them roles` --> `To learn more about inviting a user and giving them roles`. _This page_ is where you invite a user and give them roles. The linked page is where you learn more about doing it.

(Resurfaced and adapted from [earlier Slack conversation](https://gsa-tts.slack.com/files/lennybogdonoff/F5VDG5K9D/screen_shot_2017-06-15_at_6.48.53_pm.png).)